app/controllers/simplified-account/settings/team-members/change-permission/change-permission.controller.test.js

```diff
@@ -0,0 +1,91 @@
+const proxyquire = require('proxyquire')
+const sinon = require('sinon')
+const { expect } = require('chai')
+const User = require('../../../../../models/User.class')
+const userFixtures = require('../../../../../../test/fixtures/user.fixtures')
```

Comment:
might as well use the module aliases, `@test` should work now as well

Reply:
good point, done